Frends.LDAP.SearchObjects/Frends.LDAP.SearchObjects/Definitions/Input.cs

```diff
@@ -51,19 +51,29 @@ public class Input
     public SearchDereference SearchDereference { get; set; }
 
     /// <summary>
-    /// The maximum number of search results to return for a search request.
+    /// The maximum number of search results to return. 
+    /// This acts as a hard limit—regardless of page size or batch size, no more than this number of results will be returned.
+    /// Set to 0 for no limit.
     /// </summary>
     /// <example>1000</example>
     [DefaultValue(1000)]
     public int MaxResults { get; set; }
 
     /// <summary>
     /// The number of results to return in a batch. Specifying 0 means to block until all results are received. Specifying 1 means to return results one result at a time.
+    /// This does not control how many results the server fetches at once, but how results are grouped when returned.
```

Comment:
Lets change this to "This parameter controls the chunk size in which data will be read from the server. This does not affect the output result amount."